pages/docsprint/guides/participation.md

```diff
@@ -4,4 +4,62 @@ permalink: /docsprint/guides/participation/
 
 # Doc Sprint Participation Guide
 
-Here will be a guide to participating in a doc sprint.
+A well-organized doc sprint is designed to improve team documentation quickly and efficiently. Making sure 18F's documentation is accessible and comprehensive is critical for scaling what we do.  
+
+Documentation includes, but is not limited to:
+
+* READMEs, especially making sure that anyone can easily build/run a project in minutes (if not seconds)
+* Onboarding guides a la https://hub.18f.us/private/18f-site/onboarding/
+* Updating our [18F Guides](http://18f.github.io/guides/)
+
+**Here are some helpful materials and resources for participating in a doc sprint.**
+
+## <a name="todo"></a>Event Specific To-Do Lists
+
+Specific information and to-do list for each doc sprint:
+
+* [April 17 Onboarding Doc Sprint](../../onboarding/)
```

Comment:
Are we going to link to all of the open issues? How will people find a list of tasks they can work on?